Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,70 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
+        List<Person> _people = new List<Person>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_people' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896164)